### DISCUSSION TITLE
Eslint as default formatter

### DISCUSSION THREAD

**Discussion by LadIQe** [eslint/eslint#14145] [Q&A]
Hello, Im wondering if ESLint can be used as default formatting option in VS Code and also, which rules to apply.

Thing is, that functions are somehow problematic for ESLint, for example
`function (param1, param2, param3, param4, param5) {}`

this function stays as it is in ESLint, but prettier is more readable
```
function (
  param1,
  param2,
  param3,
  param4,
  param5
) {}
```

So is this possible with ESLint? Lets say, under 3 params, do it as ESLint, more than 3 params, do it as prettier? Or under 3 params but with code length greater then 120 do it as prettier?

Thing is, forcing prettier to not ignore eslint rules is really hard and creating exact same config for prettier based on eslint rules is just wasting time.

Any ideas?

Thank you

**Comment by mdjermanovic** [comment]
Hi @LadIQe!

We don't have a core rule to enforce line breaks between function parameters. Given our [rules policies](https://eslint.org/blog/2020/05/changes-to-rules-policies), it isn't likely we'll be adding any new formatting rules, but you can always make a custom rule and publish a [plugin](https://eslint.org/docs/developer-guide/working-with-plugins).
